prow/cluster/hook_deployment.yaml

```diff
@@ -50,6 +50,9 @@ spec:
         - name: oauth
           mountPath: /etc/github
           readOnly: true
+        - name: app
+          mountPath: /ect/github
```

Comment:
do you mean /etc/github? doesn't that conflict with oauth?

Reply:
yes, they will share the same `/ect/github` directory. Their name is different so there wont be conflict though.